sdk/lib/io/file_system_entity.dart

```diff
@@ -243,7 +243,8 @@ FileStat: type $type
  *   files and directories.
  */
 abstract class FileSystemEntity {
-  String get path;
+  String get path => utf8.decode(rawPath);
```

Comment:
Since the path is essentially final, we should cache the result of utf8.decode instead of recomputing it every time.

Reply:
done